drivers/haptics/drv2605.c

```diff
@@ -458,6 +459,25 @@ static int drv2605_hw_config(const struct device *dev)
 		return ret;
 	}
 
+	ret = i2c_reg_write_byte_dt(&config->i2c, DRV2605_REG_RATED_VOLTAGE, config->rated_voltage);
+	if (ret < 0) {
+		return ret;
+	}
+
+	ret = i2c_reg_write_byte_dt(&config->i2c, DRV2605_REG_OVERDRIVE_CLAMP_VOLTAGE,
+				    config->overdrive_clamp_voltage);
+	if (ret < 0) {
+		return ret;
+	}
+
+	if (config->actuator_mode == DRV2605_ACTUATOR_MODE_LRA) {
+		ret = i2c_reg_update_byte_dt(&config->i2c, DRV2605_REG_CONTROL3,
+					     DRV2605_LRA_OPEN_LOOP, DRV2605_LRA_OPEN_LOOP);
+		if (ret < 0) {
+			return ret;
+		}
+	}
```

Comment:
should this set `DRV2605_ERM_OPEN_LOOP` when in `MODE_ERM`?
can be a follow-up PR if this makes sense, and please ignore if it doesn't :)

Reply:
`ERM_OPEN_LOOP` is set by default at power-on for the DRV2605 so I think it is safe to ignore for now. Once calibration support is added, the driver will need to make a decision whether to override the default if appropriate.